config/forcedPhotCcd.py

```diff
@@ -6,6 +6,3 @@
 config.measurement.plugins["base_CircularApertureFlux"].radii = [12.0]
 
 config.measurement.slots.gaussianFlux = None
-config.doApplyExternalPhotoCalib = True
-config.doApplyExternalSkyWcs = True
-config.doApplySkyCorr = True
```

Comment:
Are `skyCorr` datasets also being bundled into `finalVisitSummary` tables? From looking through the `forcedPhotCcd` script in the [meas_base PR](https://github.com/lsst/meas_base/pull/248/files), I don't think so?

Reply:
Ah, good catch.  Backgrounds are not in visit summaries, and I think removing this last line was a mistake.